client/my-sites/plans-v2/utils.ts

```diff
@@ -240,26 +217,29 @@ export function itemToSelectorProduct(
 				comment: '%s is the name of a product',
 			} ),
 			term: item.term,
-			features: [],
-			subtypes: [],
+			features: { items: [] },
 		};
 	} else if ( objectIsPlan( item ) ) {
 		const productSlug = item.getStoreSlug();
 		let monthlyProductSlug = undefined;
 		if ( item.term === TERM_ANNUALLY ) {
 			monthlyProductSlug = getMonthlyPlanByYearly( productSlug );
 		}
-		const iconAppend = JETPACK_RESET_PLANS.includes( productSlug ) ? '_v2' : '';
+		const isResetPlan = JETPACK_RESET_PLANS.includes( productSlug );
+		const iconAppend = isResetPlan ? '_v2' : '';
+		const type = JETPACK_SECURITY_PLANS.includes( productSlug ) ? ITEM_TYPE_BUNDLE : ITEM_TYPE_PLAN;
```

Comment:
Perhaps can we store a list of bundles somewhere in `constants`? It would be easier to add bundle moving forwards if need be.